src/components/frame/v5/pages/BalancePage/partials/BalanceTable/hooks.tsx

```diff
@@ -120,44 +126,110 @@ export const useBalanceTableColumns = (
   return columns;
 };
 
-export const useGetTableMenuProps = () =>
-  useCallback<
+export const useGetTableMenuProps = (
+  data,
+  toggleActionSidebarBarOn: () => void,
+  toggleAddFundsModalOn: () => void,
+) => {
+  const [selectedToken, setSelectedToken] = useState<string>();
+  const { setTokenAddress } = useTokenContext();
+
+  useEffect(() => {
+    if (!selectedToken) return;
```

Comment:
This return should be wrapped in curly brackets {} as per
https://github.com/JoinColony/ColonyFrontEndLivingStandard/blob/master/codeQuality/index.md#explicitly-scoping-if-statements